usaspending_api/common/etl/spark.py

```diff
@@ -610,7 +610,7 @@ def write_csv_file(
     ).csv(
         path=parts_dir,
         header=False,
-        nullValue=None,
+        nullValue='',  # '' will create the output of ,,, for null values. Using None will default to ,"","",""
```

Comment:
Might need to do some regression testing to see if this affects other spark jobs like copying DBR -> CSV -> Postgres

Reply:
We plan on it. However, as long as it matches the old it's in the clear. I say that because the old covid download does not import into Databricks correctly for a couple records.

Reply:
For those same records the new covid download won't either.